Nimble/Matchers/MatcherProtocols.swift

```diff
@@ -1,5 +1,3 @@
-import Foundation
```

Comment:
`NSObject`, `NSArray`, `NSSet`, `NSHashTable`, `NSDictionary`, `NSNumber` and `NSString` are used.